docs/sources/developers/http_api/access_control.md

```diff
@@ -527,11 +527,60 @@ Content-Type: application/json; charset=UTF-8
 | 403  | Access denied.                                                       |
 | 500  | Unexpected error. Refer to body and/or server logs for more details. |
 
+### List your permissions
+
+`GET /api/access-control/users/permissions`
+
+Lists the permissions of the signed in user.
+
+#### Required permissions
+
+No permission is required.
+
+#### Query parameters
+
+| Param  | Type    | Required | Description                                                                                        |
+| ------ | ------- | -------- | -------------------------------------------------------------------------------------------------- |
+| scoped | boolean | No       | A flag indicating if we want the scopes of the permissions on top of the user's available actions. |
```

Comment:
What does "on top of the user's available actions" mean here? What does the request look like without this parameter?

Reply:
without `scoped`, you only got users actions without knowing on what they applied. But I'm removing this Query param in the end :)

Reply:
{
  "dashboards:read": true,
  "dashboards:write":  true,
  "datasources.id:read":  true,
  "datasources:read": true,
  "datasources:explore": true,
  "datasources:query":  true,
  "datasources:read":  true,
  "orgs:read": true
}